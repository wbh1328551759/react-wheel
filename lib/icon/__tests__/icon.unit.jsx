import renderer from 'react-test-renderer'
import React from 'react'
import Icon from '../icon'
import {mount} from 'enzyme'

describe('icon', () => {
  it('render successfully', () => {
    const json = renderer.create(<Icon name="alipay"/>).toJSON()
    expect(json).toMatchSnapshot()
  })
  // it('onClick', () => {
  //   const fn = jest.fn()
  //   const fn2 = jest.fn()
  //   console.log(1)
  //   console.log(mount(<button>1</button>))
  //   const component = mount(<Icon name="wechat" onClick={fn}/>)
  //   component.find('svg').simulate('click')
  //   expect(fn2).toBeCalled()
  // })

})
