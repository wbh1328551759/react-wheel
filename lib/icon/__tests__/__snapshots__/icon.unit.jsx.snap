// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon render successfully 1`] = `
<svg
  className="wr-icon"
>
  <use
    xlinkHref="#alipay"
  />
</svg>
`;
